app.py

```diff
@@ -142,7 +142,7 @@ def index_separate_iframes():
         # serve up the auth iframe to start
         request_token = ''.join(random.choice(string.lowercase + string.digits) for i in range(15))
         session['ToopherRequestToken'] = request_token
-        auth_iframe_url = api.auth_uri(username, reset_email, action, automation_allowed, challenge_required, request_token, requester_metadata, ttl);
+        auth_iframe_url = api.get_auth_url(username, reset_email, request_token, action, requester_metadata, automation_allowed=automation_allowed, challenge_required=challenge_required, ttl=ttl);
```

Comment:
Yes I think it would be good to have 'iframe' in the method name, making it clearer what it's used for and easier to search for. Should we do the same with get_user_management_url?